src/Compilers/Core/Portable/MemberDescriptor.cs

```diff
@@ -142,6 +163,9 @@ private static void ParseType(ImmutableArray<byte>.Builder builder, Stream strea
                         throw ExceptionUtilities.UnexpectedValue(typeCode);
 
                     case SignatureTypeCode.TypeHandle:
+                        ParseTypeId(builder, stream);
```

Comment:
Consider renaming to "ParseTypeHandle" to be consistent with ParseGenericTypeInstance.